examples/interpreter/physl.cpp

```diff
@@ -513,9 +513,30 @@ void print_newick_tree(std::string const& code_source_name,
 
 void print_performance_profile(
     phylanx::execution_tree::compiler::function_list& snippets,
-    std::string const& code_source_name, std::string const& dot_file,
-    std::string const& newick_tree_file, std::string const& counter_file)
+    std::string const& code_source_name, std::string const& _dot_file,
+    std::string const& _newick_tree_file, std::string const& _counter_file)
 {
+    std::int64_t _locality =
+        hpx::naming::get_locality_id_from_id(hpx::find_here());
+
+    std::ostringstream cf;
+    cf << _counter_file;
+    if(_locality > 0)
+        cf << "." << _locality;
```

Comment:
Using an `ostringstream is a bit overkill for this situation, I think. Could you just concatenate like:
```suggestion
    if(_locality > 0)
        _counter_file += std::to_string(_locality);
```